generators/server/templates/src/main/java/package/service/mapper/_UserMapper.java

```diff
@@ -54,7 +53,7 @@ default User userFromId(<% if (databaseType == 'mongodb' || databaseType == 'cas
     }
 
     default Set<Authority> authoritiesFromStrings(Set<String> strings) {
-        return strings.stream().map(string -> {
+        return strings == null ? null : strings.stream().map(string -> {
```

Comment:
Sorry I forgot to say that I'm using the entity generator.
I'm creating an EntityA that has a many-to-many relationship to a User.
And I want to be able to create an EntityA and link it to several using the generated backend and frontend code.